Sources/FeaturevisorTypes/Types.swift

```diff
@@ -116,8 +119,8 @@ public enum ConditionValue: Codable {
     case double(Double)
     case boolean(Bool)
     case array([String])
+    case unknown
     // @TODO: add Date type?
-    // @TODO: add `null` and `undefined` somehow
```

Comment:
There is no `undefined` in JSON Object at all.
If anyone would like to add any, valid JSON object should have converted it to `null`